app/src/test/java/org/simple/clinic/registration/pin/RegistrationPinScreenControllerTest.kt

```diff
@@ -81,16 +88,19 @@ class RegistrationPinScreenControllerTest {
     uiEvents.onNext(RegistrationPinTextChanged("123"))
     uiEvents.onNext(RegistrationPinDoneClicked())
 
-    verify(screen).showIncompletePinError()
     verify(userSession, never()).saveOngoingRegistrationEntry(any())
+    verify(screen).showIncompletePinError()
     verify(screen, never()).openRegistrationConfirmPinScreen()
```

Comment:
Since we have `verifyNoMoreInteractions` for the screen, we can perhaps remove this?

Reply:
We should keep this explicitly since this is something we do not want to happen. If we remove the `verifyNoMoreInteractions` line in future for whatever reason, we will have to remember to add this back and I'm sure this will be missed.

Reply:
Basically, anything we are asserting as part of the test itself should be explicit even if it's covered by a `verifyNoMoreInteractions` call.

Reply:
Got it 👍